crates/apub/src/protocol/objects/group.rs

```diff
@@ -50,6 +50,8 @@ pub struct Group {
   pub(crate) sensitive: Option<bool>,
   // lemmy extension
   pub(crate) moderators: Option<ObjectId<ApubCommunityModerators>>,
+  // lemmy extension
+  pub(crate) posting_restricted: Option<bool>,
```

Comment:
Up to you, but I would also be explicit about this: `posting_restricted_to_mods`